nvtabular/loader/backend.py

```diff
@@ -169,12 +169,18 @@ def __init__(
         shuffle,
         parts_per_chunk=1,
         devices=None,
+        global_size=None,
+        global_rank=None,
+        indices=None,
```

Comment:
This change with indices doesn't look like it's used in this PR. Seems like it might be a good idea generally, but I'm not sure if we need it here. What would be an example of when we'd use this?

Reply:
So I was thinking of adding that just in case you needed to force certain processes to run certain indices... But I dont really see the need anymore. I will pull indices out.

Reply:
actually sorry I just realized why I had that in there. Its for shuffling. We want to shuffle the indices only once (ideally on process 0) and then broadcast that out as a variable. That was a good catch. I totally forgot to add logic for testing this.

Reply:
Does our distributed seed/permutation generating scheme change this at all? Still seems useful (I think), just want to raise the question.